---a/net/minecraft/enchantment/EnchantmentArrowDamage.java
+++b/net/minecraft/enchantment/EnchantmentArrowDamage.java
@@ -2,29 +2,25 @@
 
 import net.minecraft.util.ResourceLocation;
 
-public class EnchantmentArrowDamage extends Enchantment
-{
+public class EnchantmentArrowDamage extends Enchantment {
+
     private static final String __OBFID = "CL_00000098";
 
-    public EnchantmentArrowDamage(int enchID, ResourceLocation enchName, int enchWeight)
-    {
+    public EnchantmentArrowDamage(int enchID, ResourceLocation enchName, int enchWeight) {
         super(enchID, enchName, enchWeight, EnumEnchantmentType.BOW);
         this.setName("arrowDamage");
     }
 
-    public int getMinEnchantability(int enchantmentLevel)
-    {
+    public int getMinEnchantability(int enchantmentLevel) {
         return 1 + (enchantmentLevel - 1) * 10;
     }
 
-    public int getMaxEnchantability(int enchantmentLevel)
-    {
+    public int getMaxEnchantability(int enchantmentLevel) {
         return this.getMinEnchantability(enchantmentLevel) + 15;
     }
 
-    public int getMaxLevel()
-    {
+    public int getMaxLevel() {
         return 5;
     }
-}
+}
